e2e-tests/src/vc-flow.spec.ts

```diff
@@ -22,32 +23,42 @@ test('verifieable credentials flow works end to end', async ({
   await issuerPage.goto(ISSUER_URL);
   await expect(issuerPage).toHaveTitle(/Verifiable Credentials Playground/);
 
-  expect(issuerPage.getByTestId('issuer-center-route')).not.toBeVisible();
+  expect(issuerPage.getByTestId("issuer-center-route")).not.toBeVisible();
   await issuerPage.getByTestId("go-issuer-center").click();
-  await expect(issuerPage.getByTestId('issuer-center-route')).toBeVisible();
+  await expect(issuerPage.getByTestId("issuer-center-route")).toBeVisible();
 
-  await expect(issuerPage.getByTestId('login-button')).toBeVisible();
-  const issuerAnchor = await signInWithNewUser({ page: issuerPage, context: issuerContext });
-  await expect(issuerPage.getByTestId('login-button')).not.toBeVisible();
+  await expect(issuerPage.getByTestId("login-button")).toBeVisible();
+  const issuerAnchor = await signInWithNewUser({
+    page: issuerPage,
+    context: issuerContext,
+  });
+  await expect(issuerPage.getByTestId("login-button")).not.toBeVisible();
 
   // Wait until nickname modal to be visible.
-  await expect(issuerPage.locator('[data-testid=modal]')).toBeVisible();
+  await expect(issuerPage.locator("[data-testid=modal]")).toBeVisible();
 
   // Add Issuer nickname
   const issuerName = `Issuer ${issuerAnchor}`;
-  await issuerPage.locator('input[name=prompt]').fill(issuerName);
-  await issuerPage.locator('button[type=submit]').click();
+  await issuerPage.locator("input[name=prompt]").fill(issuerName);
+  await issuerPage.locator("button[type=submit]").click();
 
-  await expect.poll(() => issuerPage.getByTestId('page-title').textContent()).toBe(issuerName);
+  await expect
+    .poll(() => issuerPage.getByTestId("page-title").textContent())
+    .toBe(issuerName);
 
-  await expect(issuerPage.getByTestId('open-create-credential-modal')).toBeEnabled();
-  await issuerPage.getByTestId('open-create-credential-modal').click();
+  await expect(
+    issuerPage.getByTestId("open-create-credential-modal")
+  ).toBeEnabled();
+  await issuerPage.getByTestId("open-create-credential-modal").click();
 
-  const credentialName = "Works at DFINITY";
+  // "Verified Humanity" is the credential that doesn't require input and therefore is backwards compatible with the rest of the flow.
+  const credentialName = "Verified Humanity";
```

Comment:
This is the only change here.

The rest are changes made by the linter. It seems that this file was never linted before 🙈